.travis.yml

```
@@ -0,0 +1,17 @@
+language: python
+python:
+  - "3.6"
+install:
+  - pip install tox docker-compose
```

You are already installing `tox` inside the Dockerfile 👍 

I guess this should be enough:
```suggestion
  - pip install docker-compose
```

---

I'm using tox without docker in travis. Thats why i am installing tox here.